docs/user/exercises/programming-exercise-setup.inc

```diff
@@ -310,13 +310,13 @@ Exercise Creation
 
   - You can change between the different repos and submit the code when needed
 
-- **Alternative 3:** Use IntelliJ with the Orion plugin and change the code directly in IntelliJ
-
   **Edit in Editor**
 
   .. figure:: programming/instructor-editor.png
             :align: center
 
+- **Alternative 3:** Use IntelliJ with the Orion plugin and change the code directly in IntelliJ
```

Comment:
The ordering of the bullet points is a bit strange in the whole file. I think the creator wanted to avoid deeply nested lists and therefore put some bullets a level higher than they belong - like the three alternatives, which share the same level as the surrounding info. While this not optimal, I did not find a better solution